src/components/home/home.component.jsx

```diff
@@ -0,0 +1,214 @@
+        <div className={classes.col3}>
+          <Title>Open and low-cost payments network for everyone.</Title>
+          <p>Hermez is a trustless zk-rollup focused on scaling payments and token transfers on the wings of Ethereum.</p>
+          <a href='/' target='_blank' rel='noopener noreferrer' className={`${classes.buttonLink} ${classes.primary}`}>
+            Create account
+            <ArrowPrimary className={classes.buttonLinkIcon} />
+          </a>
+        </div>
+      </section>
+      <section className={`${classes.section} ${classes.section2}`}>
+        <Title>Hermez seamlessly integrates into the fabric of Ethereum ecosystem and enables low-cost token transfers for inclusive economy.</Title>
+        <div className={classes.row}>
+          <div className={classes.col3}>
+            <Section21 className={classes.section2Image1} />
+            <div className={classes.section2Text}>Cost efficient token transfers and swaps with high throughput.</div>
+          </div>
+          <div className={classes.col3}>
+            <Section22 className={classes.section2Image2} />
+            <div className={classes.section2Text}>Decentalised and open-source architecture.</div>
+          </div>
+          <div className={classes.col3}>
+            <Section23 className={classes.section2Image3} />
+            <div className={classes.section2Text}>Computational integrity for secure transactions.</div>
+          </div>
+        </div>
+        <div>
+          <Link to='/payments-network' className={`${classes.buttonLink} ${classes.secondary}`}>
+            Hermez Network
+            <ArrowSecondary className={classes.buttonLinkIcon} />
+          </Link>
+        </div>
+      </section>
+      <section className={`${classes.section} ${classes.section3}`}>
+        <div className={classes.row}>
+          <div className={classes.col2}>
+            <Title>Prosperitity for all</Title>
+            <p className={classes.section3Text}>Hermez mission is to create an inclusive, secure and borderless payments infrastructure to bring economic freedom to people all over the world.</p>
+            <a href='/' target='_blank' rel='noopener noreferrer' className={classes.link}>
+              Values and guiding principles
+              <ArrowCirclePrimary className={classes.linkIcon} />
+            </a>
+          </div>
+          <div className={classes.col2}>
+            <Section31 />
+          </div>
+        </div>
+      </section>
+      <section className={`${classes.section} ${classes.section4}`}>
+        <div className={classes.row}>
+          <div className={classes.col3}>
+            <Title>Zero-knowledge cryptography</Title>
+          </div>
+          <div className={classes.col2}>
+            <p>ZK-Rollups are one of the options being developed for layer 2 construction that increases scalability through mass transfer processing rolled into a single transaction.</p>
+            <br />
```

Comment:
I prefer to use margins than brs

Reply:
Element removed, added class and corresponding styling.